src/otp-input/otp-input.tsx

```diff
@@ -125,11 +127,15 @@ export const OtpInput = ({
         if (!event.clipboardData) return;
 
         const pastedValue = event.clipboardData.getData("text");
-        const pastedValueArr = pastedValue.split("");
-        if (pastedValue && validateUserInput(pastedValue, numOfInput)) {
-            setOtpValues(pastedValueArr);
+        const valueWithoutPrefix = pastedValue.replace(prefix + "-", "");
```

Comment:
this will become `undefined-` when prefix is not specified, to check if prefix is truthy before runnign the replacement

Reply:
actually can the `-` be part of the prefix too? or are we assuming `-` is always the separator

Reply:
ah found an issue.
prefix with OTP = `abc-12356`
when we display, aesthetically it's more pleasing if it's displayed `abc - 123456` (with spaces, and the numbers are boxes)